pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -223,7 +223,53 @@ func (m *kubeGenericRuntimeManager) Status() error {
 // specifies whether the runtime returns all containers including those already
 // exited and dead containers (used for garbage collection).
 func (m *kubeGenericRuntimeManager) GetPods(all bool) ([]*kubecontainer.Pod, error) {
-	return nil, fmt.Errorf("not implemented")
+	pods := make(map[kubetypes.UID]*kubecontainer.Pod)
```

Comment:
One of the cons of CRI: this function will incur two `docker ps` calls. We'll need to swee how much performance impact there is.